widget/entry.go

```diff
@@ -836,7 +844,12 @@ func (e *Entry) eraseSelection() {
 	e.CursorRow, e.CursorColumn = e.rowColFromTextPos(posA)
 	e.selectRow, e.selectColumn = e.CursorRow, e.CursorColumn
 	e.selecting = false
-	e.updateText(provider.String())
+	changed := e.updateText(provider.String())
+
+	e.Validate()
```

Comment:
This change also leads to verifying twice (potentially running `OnChanged` twice as well, I would assume) when selecting something and then pressing another key using `TypedKey` or `TypedRune` to replace the text.

Reply:
When erasing selected text by typing the callback is ignored, so I felt confident using the OnChanged here, however looking again at where `eraseSelection` gets called, it looks like the only place that doesn't then call the callback itself is when pressing backspace or delete while in selecting mode? not sure how best to handle that.